contract-testing/spring-cloud-contract-messaging/build.gradle.kts

```diff
@@ -0,0 +1,4 @@
+subprojects {
+    group = "ws.cnt.ct.scc"
```

Comment:
Should be another group id, like `ws.cnt.ct.sccm`. Otherwise it might collide with the artifacts from the  SCC http examples when publishing the stubs into the local repository.

Reply:
done